<launch>
	<node pkg="pr_pick_pass" name="pick_pass_node" type="pick_pass_node">
		<param name="FREQ" value="100"/>
		<param name="POW_LOWER" value="-70"/>
		<param name="POW_RAISE" value="80"/>
		<param name="POW_WIND" value="120"/>
		<param name="DEG_1" value="-140"/>
		<param name="DEG_2" value="-70"/>
		<param name="DELAY_SOL" value="50"/>
		<param name="DELAY_HAND" value="500"/>
		<param name="DELAY_WIND" value="150"/>
	</node>
	<node pkg="rosserial_python" type="serial_node.py" name="serial_node0">
		<param name="port" value="/dev/ttyACM0"/>
	</node>
</launch>
